### PR TITLE
feat: vertical additional space, header slot, scroll top and bottom event, linter added, removed less

### PR DESCRIPTION
**Pull request by johndcarmichael**
- containerStyle is now the calculated height, plus the optionally injected spacer height
- header slot added sitting within the vue-virtual-collection but out of the vue-virtual-collection-container 
- the vue-virtual-collection scroll at top and bottom event triggered allowing for pagination (infinite scroll) and fetch latest to occur
- less removed as the css is trivial thus less is overkil
- eslint added to ensure styles are uniform
- package.json scripts updated
- position by translateX and Y opposed to position top/left

### CODE REVIEW

**Comment by johndcarmichael** [comment]
There appears to be a bug when using the header slot - as the slot pushes down the content, the in view measure meant of the cards is at times off in chrome. In FF however no issue :/

**Comment by johndcarmichael** [comment]
@starkwang any chance of getting this PR reviewed soon, could do with it published to npm for me CI pipes :)

**Comment by starkwang** [comment]
> @starkwang any chance of getting this PR reviewed soon, could do with it published to npm for me CI pipes :)

I'll review this PR ASAP.

File: README.md [hunk 1]
```diff
@@ -236,24 +265,77 @@ Type: `number`
...
+
+Type: `number`
+
+Optionally extend the calculated height of the collection container, the affect is apparent padding-bottom
+
+### headerSlotHeight
+
+Type: `number`
+
+When injecting content into the header slot, you should also add the px height of the slot. This ensure items are removed from view at the right time.
```
Comment (starkwang) [inline README.md:295]: Could you note default value of these props?

Comment (johndcarmichael) [reply README.md:298]: updated

**Comment by starkwang** [comment]
Thanks for your contribution! @johndcarmichael

**Comment by starkwang** [comment]
Landed in 1.5.0

**Pull request merged**